nifi-assembly/pom.xml

```diff
@@ -322,6 +322,10 @@ language governing permissions and limitations under the License. -->
         <nifi.administrative.yield.duration>30 sec</nifi.administrative.yield.duration>
         <nifi.bored.yield.duration>10 millis</nifi.bored.yield.duration>
 
+        <nifi.processor.rollback.count.enabled>false</nifi.processor.rollback.count.enabled>
+        <nifi.processor.rollback.log.unack.ff.enabled>false</nifi.processor.rollback.log.unack.ff.enabled>
+        <nifi.processor.rollback.log.unack.ff.max>5</nifi.processor.rollback.log.unack.ff.max>
```

Comment:
I do think it would be great if we can find a nice way to signal flowfiles recently involved in a rollback.  But I think whatever the solution is it should not require additional configuration/thought on the admin's part.